### ISSUE TITLE
Bug with domain "pbr.digital"

### ISSUE THREAD

**Issue by amunhoz** [joshterrill/freewhois#3]
You can fix with this

```

function findRDAPUrl(domain) {
    if (!domain) {
        throw new Error("You must enter a domain");
    }
    const tld = domain.split(".")[domain.split(".").length -1];
    if (!tld || tld === "") {
        throw new Error("Error parsing domain");
    }

    if (!dns.length) {
        const dnsFile = fs.readFileSync(`${__dirname}/tlds.json`, "utf-8");
        dns = JSON.parse(dnsFile);
    }
     /********************************/
     //find other itens in array
    /********************************/
    const foundTld = dns.find((i) =>{ 
        for (var ix in i[0]) {
            if (i[0][ix]==tld) return true //find all itens in array
        }
        return false
    });
  

    if (!foundTld) {
        throw new Error(`Unable to find tld ${tld}`);
    }
  

    /********************************/
   //remove last slash
    /********************************/
    const rdapUrl = foundTld[1][0].replace(/\/$/, ""); 
    return rdapUrl;
}
```

**Comment by joshterrill** [comment]
Got it. It looks like some of the other domains are put in a list rather than each single domain being added as separate records. I'll have a fix for it soon.

**Comment by joshterrill** [comment]
@amunhoz I have fixed this in my latest commit. I took a slightly different syntax approach than what you recommended, but the idea is the same. Feel free to look at the commit here: https://github.com/joshterrill/freewhois/commit/89b7dfebca66b6cda78d49b050f57a9fcf4b0381 I also created a unit test for your exact scenario and published a new npm version, which is version `1.2.0`
